crates/swc_ecma_utils/src/lib.rs

```diff
@@ -1375,18 +1375,29 @@ pub trait ExprExt {
                         }
                     }
                     Expr::Object(obj) => {
-                        let is_static_accessor = |prop: &PropOrSpread| {
-                            if let PropOrSpread::Prop(prop) = prop {
-                                if let Prop::Getter(_) | Prop::Setter(_) = &**prop {
-                                    true
-                                } else {
-                                    false
-                                }
-                            } else {
-                                false
+                        let is_prop_could_has_side_effect = |prop: &PropOrSpread| {
```

Comment:
Can you rename this to `can_have_side_effect`?

Reply:
Done.  Could you re-run the wasm test? I saw it failed due to 
> Run curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
sh: 139: [: x86_64: unexpected operator
info: downloading wasm-pack
info: successfully installed wasm-pack to `/home/runner/.cargo/bin/wasm-pack`
Segmentation fault (core dumped)
Error: Process completed with exit code 1.